vault-fetcher/fetch_secrets.py

```
@@ -0,0 +1,81 @@
+import os
+import sys
+import signal
+from supabase import create_client, Client
```

Ruff (0.11.9)

`supabase.create_client` imported but unused

Remove unused import

(F401)